nalgebra-lapack/tests/linalg/generalized_eigenvalues.rs

```diff
@@ -0,0 +1,74 @@
+use na::dimension::{Const, Dynamic};
+use na::{DMatrix, EuclideanNorm, Norm, OMatrix};
+use nl::GE;
+use num_complex::Complex;
+use simba::scalar::ComplexField;
+use std::cmp;
+
+use crate::proptest::*;
+use proptest::{prop_assert, proptest};
+
+proptest! {
+    #[test]
+    fn ge(n in PROPTEST_MATRIX_DIM) {
+        let n = cmp::max(1, cmp::min(n, 10));
+        let a = DMatrix::<f64>::new_random(n, n);
+        let b = DMatrix::<f64>::new_random(n, n);
+        let a_condition_no = a.clone().try_inverse().and_then(|x| Some(EuclideanNorm.norm(&x)* EuclideanNorm.norm(&a)));
```

Comment:
This is not a very numerically stable way of computing the condition number of a matrix. It might be OK since you anyway just filter out large numbers though.

Also, instead of using an `if` condition below, consider using proptest's facilities for rejecting the test input. Then it will warn you if it rejects too many inputs. Otherwise you risk accidentally rejecting almost *all* input without ever knowing! Since you ask for matrices with condition number < 5 it seems likely you're rejecting a great deal of matrices, so the usefulness of the test is strongly tied to the distribution of matrices generated by `matrix()`, which is

Reply:
Sorry I didn't finish the above comment: that's the point I realized that matrices are generated by `new_random`, not `matrix()`!